### ISSUE TITLE
*Feature request*: add webgl support for copy v86

### ISSUE THREAD

**Issue by SugarRayLua** [copy/v86#756]
Please consider adding webgl support for copy v86 by adding:
xterm-addon-webgl package to xterm.js to copy v86

Would like to see such support and how that might work in the ArchLinux demo

Thanks!

**Comment by copy** [comment]
Note that xterm only renders the serial console, not the graphical display.

I don't really see the point of rendering text with webgl, and the [readme](https://github.com/xtermjs/xterm.js/blob/master/addons/xterm-addon-webgl/README.md) doesn't explain it either, but I can see plenty of downsides. Unless there's a strong argument, I'd vote to close this issue.

**Comment by SugarRayLua** [comment]
Thanks, @copy. I'm a novice Linux user so please bear with me if I'm not understanding something correctly, but don't xclock and xeyes both require a graphical display? If so, xterm in the Arch Linux demo in the copy emulator displays xclock well in addition to xeyes (although on Arch Linux, I had to download the package xeyes first). That is why I thought xterm in copy was rending a graphical display and why then the the addition of the webgl support through the xterm-addon-webgl package would be a good addition to the copy emulator. If I'm misunderstanding this, then I definitely agree that adding webgl would not be useful and apologize for misunderstanding xterm. Thanks!

**Comment by SugarRayLua** [comment]
Sorry, @copy, I think I researched more and answered my own question and understand better your concern about the webgl add on. It's the X11 server itself that is supposed to handle graphics and openGL graphics calls (including ones that may later get translated to webGL in a browser) and that xterm's function is just to display a serial text console within an X11 graphical environment. If so, I then understand your question of why would having webgl support for an xterm be helpful? I was thinking it would add webGL support to the X11 environment inside of Linux running inside the copy emulator. Thus, I wrote the xterm.js developer to ask him if he could reply and explain more what the webgl add-on package for xterm.js is supposed to accomplish.
